data_io/location.py

```diff
@@ -10,10 +10,11 @@ class Location(URL):
     stub for extension
     """
 
-    def __init__(self, *largs, **kwargs):
+    def __init__(self, url, *largs, **kwargs):
```

Comment:
I don't understand this change

Reply:
Initially was going to have default port's in location